src/figuresClasses.ts

```diff
@@ -1,11 +1,78 @@
-export interface Figure {}
+export interface Figure {
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
 
-export class Triangle implements Figure {}
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  public shape: 'triangle';
+
+  constructor(
+    public color: 'red' | 'green' | 'blue',
```

Comment:
```suggestion
    public color: Color,
```
and here you can just use Cololr type